src/pe_reports/report_generator.py

```diff
@@ -265,12 +260,11 @@ def main():
         os.mkdir(validated_args["OUTPUT_DIRECTORY"])
 
     # Generate reports
-    generated_reports = generate_reports(
+    generate_reports(
         validated_args["REPORT_DATE"],
         validated_args["OUTPUT_DIRECTORY"],
+        validated_args["--soc_med_included"],
```

Comment:
This instance of `--soc_med_included` needs to be updated as well:
```suggestion
        validated_args["--soc-med-included"],
```